applications/coreutils/yes.cpp

```diff
@@ -0,0 +1,40 @@
+#include <libsystem/io/Stream.h>
+#include <libutils/StringBuilder.h>
+
+String concat(int argc, char **argv)
+{
+    int i;
+    StringBuilder builder;
+
+    for (i = 1; i < argc - 1; i++)
```

Comment:
I'm not sure about the -1 here. Are we not missing the last arguments.

Reply:
No, because I add it at line 15

Reply:
(I did a -1 only to don't have a space at the end)

Reply:
Sorry, I need to improved at reading other people code 😅 Good work :)

Reply:
No problem :sweat_smile: